README.adoc

```diff
@@ -1,101 +1,178 @@
 = Asciidoctor Diagram
-Pepijn Van_Eeckhoudt
-
-Asciidoctor Diagram is a set of extensions for http://asciidoctor.org[Asciidoctor], the Ruby-based AsciiDoc processor.
-These extensions allow you to embed plain text diagrams inside your AsciiDoc documents using one of the following syntaxes:
-
-- http://blockdiag.com[BlockDiag, SeqDiag, ActDiag, NwDiag]
-- http://ditaa.sourceforge.net[Ditaa].
-- http://www.graphviz.org/content/dot-language[GraphViz DOT]
-- http://knsv.github.io/mermaid/[Mermaid]
-- http://plantuml.sourceforge.net[PlantUML]
-- https://github.com/christiangoltz/shaape[Shaape]
-- http://wavedrom.com[WaveDrom]
-
-Additionally Asciidoctor Diagram includes a basic meme generator extension.
-
-The extension takes care of running the diagram processor to generate the images from the input text and insert them into the rendered document.
-
-This gem was inspired by the https://code.google.com/p/asciidoc-plantuml/[AsciiDoc PlantUML filter] for AsciiDoc Python.
-
+Pepijn Van_Eeckhoudt <https://github.com/pepijnve[@pepijnve]>; Sarah White <https://github.com/graphitefriction[@graphitefriction]>
+:description: README for the Asciidoctor Diagram extension for Asciidoctor.
+:toc: preamble
+ifdef::env-github[:toc: macro]
+:icons: font
+:source-highlighter: coderay
+:source-language: asciidoc
+:table-caption!:
+:example-caption!:
+:figure-caption!:
+:check: icon:check[]
+ifdef::env-github[:check: :ballot_box_with_check:]
+ifndef::env-site[:status:]
+:uri-actdiag: http://blockdiag.com/en/actdiag/index.html
+:uri-asciidoctor-api: http://asciidoctor.org/docs/user-manual/#api
+:uri-asciidoctor-extensions: http://asciidoctor.org/docs/user-manual/#extension-points
+:uri-blockdiag: http://blockdiag.com
+:uri-ditaa: http://ditaa.sourceforge.net/
+:uri-dot: http://www.graphviz.org/content/dot-language
+:uri-graphviz: http://www.graphviz.org
+:uri-imagemagick: http://www.imagemagick.org
+:uri-java: http://java.sun.com
+:uri-mermaid: http://knsv.github.io/mermaid/
+:uri-nwdiag: http://blockdiag.com/en/nwdiag/index.html
+:uri-packetdiag: http://blockdiag.com/en/nwdiag/index.html
+:uri-phantomjs: http://phantomjs.org
+:uri-plantuml: http://plantuml.sourceforge.net
+:uri-py-plantuml: https://code.google.com/p/asciidoc-plantuml/
+:uri-rackdiag: http://blockdiag.com/en/nwdiag/index.html
+:uri-seqdiag: http://blockdiag.com/en/seqdiag/index.html
+:uri-shaape: https://github.com/christiangoltz/shaape
+:uri-wavedrom: http://wavedrom.com
+:uri-wavedromeditor: https://github.com/wavedrom/wavedrom.github.io/releases
+:uri-wavedromcli: https://github.com/wavedrom/cli
+
+Asciidoctor Diagram is a set of extensions for Asciidoctor that enable you to add diagrams to your AsciiDoc document which you describe using plain text.
```

Comment:
I think the subject gets mixed up in this opening sentence. Consider rewording it as follows:

```
Asciidoctor Diagram is a set of Asciidoctor extensions that enable you to add diagrams, which you describe using plain text, to your AsciiDoc document.
```